sentences/nl/light_HassLightSet.yaml

```diff
@@ -11,4 +11,7 @@ intents:
           - "<doe_enkelvoud> <area> <helderheid> <naar> <brightness>"
         slots:
           name: "all"
-
+      # color
+      - sentences:
```

Comment:
Kunnen we ook `Maak <name> {color}` ondersteunen ("Maak het bolletje rood")? Zouden we `maak` bij `doe_enkelvoud` voegen of het als een losse zin hier houden ?

Reply:
Dit kan in een volgende PR